catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -545,4 +549,28 @@ public boolean isBackgroundSprite() {
 	public Multimap<EventId, EventThread> getIdToEventThreadMap() {
 		return idToEventThreadMap;
 	}
+
+	public int getUnusedPointer() {
+		int result = 0;
+		while (result < 20 && usedTouchPointer.contains(result)) {
+			++result;
+		}
+		usedTouchPointer.add(result);
+
+		return result;
+	}
+
+	public void releaseUsedPointer(int position) {
+		usedTouchPointer.removeAll(Collections.singleton(position));
```

Comment:
for removing one object from an ArrayList, `remove` should be used instead of `removeAll`, right?

Reply:
never mind, I got it, this removes all occurrences of the position from ArrayList